app/Http/Resources/AuthUserResource.php

```diff
@@ -20,7 +20,8 @@ public function toArray($request)
            'token_type'=>'Bearer',
            'user'=>[
             "id"=> $this->id,
-            "name"=> $this->name,
+            "name"=> $this->getTranslations('name'),
+            "name_translatable"=> $this->name,
```

Comment:
replace name with name translateble
name-translatable=>$this->getTranslations('name')